CHANGELOG.md

```diff
@@ -18,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
      - Enclave apps that are built with Make and rely on Open Enclave's pkgconfig must now explicitly include OE crypto wrapper library in linker dependency flags.
      - See the [Makefile in the helloworld sample](samples/helloworld/enclave/Makefile#L34) for an example. Here `OE_CRYPTO_LIB` is set to `mbedtls` in [parent MakeList file](samples/helloworld/Makefile#L9).
 
+
 ### Changed
 - Syscalls are internally dispatched directly to their implementation functions instead of via a switch-case.
```

Comment:
Please add this bullet item to **Changed** section:

- Changed the attestation evidence extension OIDs for certificates generated by the following APIs. Verifiers must call oe_verify_attestation_certificate APIs from v.0.11.0 or above.
oe_generate_attestation_certificate(): "1.3.6.1.4.1.311.105.1"
oe_get_attestation_certificate_with_evidnece(): "1.3.6.1.4.1.311.105.2"